### ISSUE TITLE
Add longopts

### ISSUE THREAD

**Issue by mmkal** [entropitor/dotenv-cli#97]
Hi - I like this library a lot, one small request. In build-scripts/any code or documentation that's checked in to source control, our team have a loose convention to always use long options (e.g. `--option`) rather than short options (e.g. `-o`) for CLI programs. The idea is that short options are helpful when you're typing and want to move fast, but they're worse if you're reading commands, (say, in an npm script), because they're harder for a newcomer or a code-reviewer to look at and guess what the option does.

So, I think it'd be worthwhile to add long options for these specific options:

```
  -e <path>           parses the file <path> as a `.env` file and adds the variables to the environment
  -e <path>           multiple -e flags are allowed
  -v <name>=<value>   put variable <name> into environment using value <value>
  -v <name>=<value>   multiple -v flags are allowed
  -p <variable>       print value of <variable> to the console. If you specify this, you do not have to specify a `command`
  -c [environment]    support cascading env variables from `.env`, `.env.<environment>`, `.env.local`, `.env.<environment>.local` files
```

I'd suggest:

- `-e` -> `--env`
- `-v` -> `--variable`
- `-p` -> `--print`
- `-c` -> `--config`

**Comment by entropitor** [comment]
Makes sense! Feel free to create a PR. It seems we already have support for this https://github.com/entropitor/dotenv-cli/blob/master/cli.js#L32
